### ISSUE TITLE
Color Input widgets error when using command center

### ISSUE THREAD

**Issue by TheBizzle** [NetLogo/Galapagos#370]
_From @mrerrormessage on March 2, 2017 15:35_

When I open CA 1D Elementary and enter any command in the command center, I receive the following error:

Invalid input box: ArrayBuffer((multiline,null), (type,"Color"), (value,55)) Invalid input box: ArrayBuffer((multiline,null), (type,"Color"), (value,0))

The error doesn't occur when *running* the model, only when using the command center. This bug also occurs on Life, which also uses two color inputs.

_Copied from original issue: NetLogo/Tortoise#182_

**Comment by TheBizzle** [comment]
_From @LaCuneta on May 23, 2017 14:29_

In reviewing, we found that the root cause was the Ractive CoffeeScript template binding the multiline value for editing on the numeric types (Color and Number both).  The fix is to change the template to only bind the value for String types, where it's appropriate.  Once it's merged, this will be fixed.  We also requested a change in NetLogo core to only allow the multiline value on String input box types.
